desktop-app/app/reducers/browser.js

```diff
@@ -237,7 +237,7 @@ export default function browser(
       ? getLastOpenedAddress()
       : getHomepage(),
     currentPageMeta: {},
-    zoomLevel: 0.6,
+    zoomLevel: _getUserPreferences().zoomLevel || 0.6,
```

Comment:
@jzabala Can you please add `zoomLevel` field to `UserPreferenceType` type above?

Reply:
Added!